traitlets/traitlets.py

```diff
@@ -2386,296 +2362,311 @@ def info_rst(self):
         return self._info(as_rst=True)
 
 
-class Container(Instance):
-    """An instance of a container (list, set, etc.)
-
-    To be subclassed by overriding klass.
-    """
-    klass = None
-    _cast_types = ()
-    _valid_defaults = SequenceTypes
-    _trait = None
-
-    def __init__(self, trait=None, default_value=None, **kwargs):
-        """Create a container trait type from a list, set, or tuple.
-
-        The default value is created by doing ``List(default_value)``,
-        which creates a copy of the ``default_value``.
+class Mutable(Instance):
 
-        ``trait`` can be specified, which restricts the type of elements
-        in the container to that TraitType.
+    def default(self, obj=None):
+        if self.default_value is not Undefined:
+            return copy.copy(self.default_value)
+        elif hasattr(self, "make_dynamic_default"):
+            return self.make_dynamic_default()
+        else:
+            return Undefined
 
-        If only one arg is given and it is not a Trait, it is taken as
-        ``default_value``:
 
-        ``c = List([1, 2, 3])``
+class Container(Instance):
+    """A base class for iterable types"""
```

Comment:
More info in the docstrings would be helpful. How would I subclass Container, for instance.